ports/servoshell/prefs.rs

```diff
@@ -530,6 +535,29 @@ pub(crate) fn parse_command_line_arguments(args: Vec<String>) -> ArgumentParsing
         })
         .collect();
 
+    if opt_match.opt_present("enable-experimental-web-platform-features") {
+        vec![
+            "dom_fontface_enabled",
```

Comment:
The actual list of features to enable can be discussed as part of the review, I have selected a few but maybe some are not good candidates yet for this and maybe I'm missing others.

Reply:
`dom_xpath_enabled` is another useful one, because it makes HTMX work better: https://github.com/servo/servo/issues/34060

Reply:
> `dom_xpath_enabled` is another useful one, because it makes HTMX work better: #34060

Added that one too.